src/main/java/org/opensearch/ad/util/RestHandlerUtils.java

```diff
@@ -81,6 +81,10 @@ public final class RestHandlerUtils {
     public static final String RESULTS = "results";
     public static final String TOP_ANOMALIES = "_topAnomalies";
     public static final String VALIDATE = "_validate";
+
+    public static final String NODE_ID = "nodeId";
+
+    public static final String STAT = "stat";
```

Comment:
We generally add the constant for rest handler URI but it's fine to add for request param as well. Can you remove the extra lines here?